ui/signed-in/build.gradle.kts

```diff
@@ -33,6 +33,9 @@ android {
         }
     }
 }
+dependencies {
+    implementation(project(mapOf("path" to ":data:persistence")))
+}
```

Comment:
What's this for?

Reply:
was studio auto adding dependency and not well. will remove